lib/hashie/extensions/coercion/active_model.rb

```diff
@@ -0,0 +1,85 @@
+require 'active_model/type'
+
+module Hashie
+  module Extensions
+    module Coercion
+      module ActiveModel
+        # Symbol is the one glaring type omission. Define it if it is not
```

Comment:
This already is Mashie's namespace: `Hashie::Extensions::Coercion::ActiveModel`. Anything to do here?

I believe there is no symbol type because the focus of ActiveModel::Type is the interop with DBs - none of which use symbols.